protocol-units/maptos-rest/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+use poem::{handler, IntoResponse, Response, Route};
+
+pub struct MaptosConfig {
+	pub url: String,
+	// other fields for configs, log verbosite etc.
+}
+
+impl MaptosConfig {
+	pub const MAPTOS_REST_ENV_VAR: &'static str = "MAPTOS_REST_URL";
+
+	pub fn try_from_env() -> Result<Self, anyhow::Error> {
+		let maptos_rest =
+			std::env::var(Self::MAPTOS_REST_ENV_VAR).unwrap_or("0.0.0.0:30832".to_string());
+		Ok(Self { url: maptos_rest })
+	}
+}
+
+#[handler]
+async fn health() -> Response {
+	"healthy".into_response()
+}
+
+#[handler]
+async fn state_root_hash() -> Response {
```

Comment:
You will want `Context` here. Are you trying to merge this PR before the actual implementation is done for organizational purposes?

Reply:
I was yeah, just to keep PRs small. But I'l just keep comitting here, till the whole e2e test is here.